src/test/java/io/percy/selenium/SdkTest.java

```diff
@@ -82,4 +82,10 @@ public void snapshotsLiveHTTPSSite() {
     driver.get("https://www.google.com");
```

Comment:
How about update the test suite to snapshot https://sdk-test.percy.dev and remove google.com here. You could borrow some of the test wording too from https://github.com/percy/percy-puppeteer/blob/master/tests/sdk_spec.js#L99

I'd become pretty convinced this is all working well if you can take a successful snapshot of our SDK test website.